packages/cli/src/credentials/credentials.service.ts

```diff
@@ -108,23 +123,38 @@ export class CredentialsService {
 				credentials = credentials.map((c) => this.ownershipService.addOwnedByAndSharedWith(c));
 			}
 
-			if (options.includeScopes) {
+			if (includeScopes) {
 				credentials = credentials.map((c) =>
 					this.roleService.addScopes(c, user, projectRelations!),
 				);
 			}
 
+			if (includeData) {
+				credentials = credentials.map((c: CredentialsEntity & ScopesField) => {
+					if (c.scopes.includes('credential:update')) {
```

Comment:
I don't have all thecontext but why do we include the data only if there is a `credential:update` scope? Do we test this case?